bin/posix/setup_posix.sh

```diff
@@ -1,2 +1,2 @@
-cmake -DOpENer_PLATFORM:STRING="POSIX" -DCMAKE_BUILD_TYPE:STRING="" -DBUILD_SHARED_LIBS:BOOL=OFF ../../source
+cmake -DCMAKE_C_COMPILER=gcc -DOpENer_PLATFORM:STRING="POSIX" -DCMAKE_BUILD_TYPE:STRING="" -DBUILD_SHARED_LIBS:BOOL=OFF ../../source
```

Comment:
Why is this change needed?

Reply:
Technically it's not needed. However, in my local setup when I first ran the AFL setup it caused cmake to remember the last compiler I used and so when I later wanted to compile the release version it used the AFL compiler by default.

To eliminate any potential confusion I specified explicitly the compiler I want to use